package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "resin-typescript-skeleton",
+  "version": "0.0.1",
+  "description": "Skeleton template for a Resin TypeScript project",
+  "homepage": "https://github.com/resin-io/resin-typescript-skeleton#readme",
+  "preferGlobal": true,
+  "types": "build/**/*.d.ts",
```

Comment:
I'll need to check this. I actually found this on a TS thread with a VisualCode developer specifying it. Will need to try and re-find that thread.